modules/aws-outposts-load-balancer-clb.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-load-balancer-clb_{context}"]
+= Using AWS Classic Load Balancers in an AWS VPC cluster extended into an Outpost
+
+Outposts racks cannot run AWS Classic Load Balancers, but Classic Load Balancers in the AWS VPC cluster can attach to edge compute resources in the Outpost.
+As a result, Classic Load Balancers on the VPC cluster might schedule pods on either of these node types.
+
+If you want to prevent the Classic Load Balancer in the VPC cluster from attaching to Outpost edge compute nodes, you can apply labels to the edge compute nodes and configure the Classic Load Balancer to avoid scheduling on nodes with the applied labels.
+
+[NOTE]
+====
+If you do not need to prevent the Classic Load Balancer in the VPC cluster from attaching to Outpost edge compute nodes, you do not need to complete these steps.
+====
+
+.Prerequisites
+
+* You have extended an AWS VPC cluster into an Outpost.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+* You have installed the AWS Load Balancer Operator and created the AWS Load Balancer Controller.
+
+.Procedure
+
+. List the nodes on the Outpost by running the following command:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                   STATUS    ROLES     AGE       VERSION
+node1.example.com      Ready     worker    7h        v1.28.5
+node2.example.com      Ready     worker    7h        v1.28.5
+----
+//Does this need some additional filtering to ensure we are only listing Outposts nodes and not the VPC nodes too?
+
+. For each, label the node with a key-value pair by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node_name> <key_name>=<value>
+----
++
+#is there a way to label them all at once in case there are a bunch? Something related to the `--all` parameter here? https://docs.openshift.com/container-platform/4.14/nodes/nodes/nodes-nodes-working.html#nodes-nodes-working-updating_nodes-nodes-working#
+
+. #How does the user find these to edit them? Or is this assuming from new?#
```

Comment:
I have 3 commands, the first one cannot check it's public or private, the second one can only list private subnet, and the third one can list all the subnet information. @yunjiang29 may have a better idea here. 
```
oc get cm -n kube-system cluster-config-v1 -o json |jq -r '.data["install-config"]' |yq .platform.aws.subnets

oc -n openshift-machine-api get machineset zhsun-out-8p8nv-worker-us-east-1a -o jsonpath="{.spec.template.spec.providerSpec.value.subnet.id}" 

aws ec2 describe-subnets \
--filters "Name=tag:kubernetes.io/cluster/<infrastructure_id>,Values=shared" \
--output json
```

Reply:
> oc -n openshift-machine-api get machineset zhsun-out-8p8nv-worker-us-east-1a -o jsonpath="{.spec.template.spec.providerSpec.value.subnet.id}" 

I think this is a good way.  For getting non-Outpost private subnet ids, we don't have a perfect way to get them, but we can get one of them from an existing compute node's (not outpost) machineset

Reply:
Sorry, I think I didn't ask this clearly (or I might misunderstand what is required. We are telling the user to edit a `deployment` resource and `service` resource for the Classic Load Balancer in the VPC cluster. 

What commands do they run so that they can see the YAML to edit it? I think probably some versions of `oc edit`, but I am not sure what the rest of the parameters would be for each resource.

After they edit the YAML, I have added some `oc create -f` steps to create the deployment and resource from the edited file, but I am not 100% sure about those steps either. Do those look ok?

Reply:
Note: I did add the step to get the VPC subnet to [Obtaining information from your OpenShift Container Platform cluster](https://71263--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/configuring-aws-outposts#aws-outposts-environment-info-oc_configuring-aws-outposts), so the user will have that information available for all the tasks that need it.

Reply:
@jeana-redhat this is from new, if user create new CLB and this could ensure the CLB lands in the regular AZ and not the outposts AZ. `oc create -f` is correct.

Reply:
Thannks, @sunzhaohua2 :tada: